### PR TITLE
Add documentation for date input and adds ability to hide vue 2 sandb…

### PR DESCRIPTION
**Pull request by benjag**
…ox link

### CODE REVIEW

File: docs/components/input/README.md [hunk 1]
```diff
@@ -436,6 +436,29 @@ Use the `type="number"` attribute only for input fields that reference a numeric
...
+  type="date"
+  @input="restrictInput"
+/>
+```
+
+</cdr-doc-example-code-pair>
+
 ### Input with Link Text
 
 Input field with link text on right. The link should describe it's relationship to the input field either through it's text content or an aria-label.
```
Comment (MegMariel) [inline docs/components/input/README.md:464]: replace "it's" with "its" (2x)

**Review by mhewson — APPROVED** [review]

File: docs/components/input/README.md [hunk 2]
```diff
@@ -436,6 +436,29 @@ Use the `type="number"` attribute only for input fields that reference a numeric
 
 </cdr-doc-example-code-pair>
 
+### Date input
+
+<cdr-banner type="info"><template #icon-left><icon-information-fill inherit-color /></template><span style="font-weight: 500">Vue 3 Only</span></cdr-banner>
+
+Use `type="date"` to create input fields that let a user enter a date, either with a validated text field and/or a simple calendar UI. The presentation is dependent on the browser and largely immutable so this is best used for applications where consistent presentation is less important than the functional benefits.
```
Comment (MegMariel) [inline docs/components/input/README.md:443]: Change paragraph to:

Use `type="date"` to create input fields which allow users to enter a date, either with a validated text field and/or a simple calendar UI. Note: the presentation is dependent on the browser and largely unchangeable. Use this input field for applications where consistent presentation is less important than the functional benefits.

File: docs/release-notes/summer-2022/README.md [hunk 3]
```diff
@@ -30,6 +30,12 @@ If your project depends on any shared component packages, you will want to updat
 
 <br />
 
+## 13.3.0
+
+Adds `type="date"` to CdrInput to support browser-native date-picking functionality. See the [CdrInput docs page](../../components/input#date-input) for more information.
```
Comment (MegMariel) [inline docs/release-notes/summer-2022/README.md:35]: Replace "docs page" with "article"

**Review by MegMariel — CHANGES_REQUESTED** [review]
Left a handful of comments suggesting small copy edits.

**Review by mhewson — APPROVED** [review]

**Pull request merged**